ScottWu.md

```diff
@@ -16,21 +16,21 @@
 
 - General features
 	- [feedback?] Passing data between input field and pickers
-	- Keyboard Controls
-		- Jessica will handle keyboard events, UI only receives changes made
+	- [WIP] Keyboard Controls
+		- Input box handles keyboard events most of the time, until user triggers shifting "visual focus" to picker
 - Month picker
 	- [Done] Spinner component (basic functionalities)
 	- [Hold] Spinners are combined when indivisible step is used
 		- UX considering removing this from the spec
 - Time picker
 	- [Done] AM/PM component
-	- [WIP] Spinner cycle feature
+	- [Done] Spinner cycle feature
```

Comment:
I tried detecting scroll stop and do append/prepend elements, but the end result is not as smooth as I hoped, and it makes regular scrolling janky sometimes. So instead of doing that, I just re-center the scroll position when we hit bottom/top and avoid adding/removing nodes altogether.